domains/integration-test/pvp_test.go

```diff
@@ -406,6 +406,9 @@ func (s *pvpTestSuite) TestNotoForZeto() {
 	})
 	lockedZeto, _ := lockedZetoCoins[0].Data.Hash(ctx)
 
+	// TODO: remove
```

Comment:
Would love to see this removed before merge - just a check-loop would be better

Reply:
yep the check-loop was already being done with the `findAvaillableCoins()` call immediately above it. it was added during debugging and isn't needed anymore. removed.